spine-godot/spine_godot/SCsub

```diff
@@ -2,10 +2,10 @@ Import('env')
 
 # Need to add the include path to env so the vsproj generator consumes it.
 if env["vsproj"]:
-    env.Append(CPPPATH=["#../spine_godot/spine-cpp/include"])
```

Comment:
All the changes in this file must be reverted.

Reply:
Will do.